content/02.delete-me.md

```diff
@@ -1,15 +1,146 @@
-## Manubot Rootstock Information
+## Introduction
 
-**Note: Manubot instances should delete this file.**
+This manuscript is a template/boilerplate (aka "rootstock") for [Manubot](https://manubot.org/ "Manubot"), a tool for writing scholarly manuscripts.
+Use this template as a starting point for your manuscript by following the directions on the [repository homepage](https://github.com/greenelab/manubot-rootstock "Manubot Rootstock").
 
-The Manubot is a system for automating scholarly publishing.
-Content is written in [Pandoc Markdown](http://pandoc.org/MANUAL.html#pandocs-markdown) source files.
-See [`USAGE.md`](https://github.com/greenelab/manubot-rootstock/blob/master/USAGE.md) for more information on how to use the Manubot.
 
-The Manubot project began with the [Deep Review](https://github.com/greenelab/deep-review), where it was used to compose a highly-collaborative review article [@doi:10.1098/rsif.2017.0387].
-Another example manuscript that was created with Manubot is:
 
-+ The Sci-Hub Coverage Study ([GitHub](https://github.com/greenelab/scihub-manuscript), [HTML manuscript](https://greenelab.github.io/scihub-manuscript/)) [@doi:10.7554/eLife.32822]
+## Examples
 
-If you notice a problem with Manubot, it's best to submit an upstream fix to the appropriate repository:
-[`greenelab/manubot-rootstock`](https://github.com/greenelab/manubot-rootstock) for the git repository stub or [`greenelab/manubot`](https://github.com/greenelab/manubot) for the Python package.
+Below is a full list of formatting elements for `.html` and `.pdf` exports that are supported by the Manubot-provided themes. 
+Compare the input Markdown (`.md` files in the `/content` directory) to the output you see below.
```

Comment:
This line assumes the viewer is viewing the file from GitHub, which is not always the case. Additionally, many of these features are not supported by GHFM (github flavored markdown). Would be better to say output at https://greenelab.github.io/manubot-rootstock/?

Reply:
I'm not sure I understand what you mean. "The output you see below" should apply to whatever output format the user is viewing at that time. If they have `manuscript.html` open, they'll see the rendered page in their browser. If they have `manuscript.pdf` open, they'll see how it looks as a pdf.

Regarding GHFM, I'm aware that a few things wont display right on github, like the `{}` parameters, citations, latex, and button - but that's why I specify that this list only applies to the `.html` and `.pdf` outputs.

Reply:
should we say "to the output you see below in the rendered manuscript".